### PR TITLE
Add 1.23 iterators support

### PR DESCRIPTION
**Pull request by errx**
Hello, I've added support for iterators, method names are taken from https://pkg.go.dev/maps package

### CODE REVIEW

File: map64.go [hunk 1]
```diff
@@ -290,6 +334,22 @@ func forEach64[K IntKey, V any](pairs []pair[K, V], f func(k K, v V) bool) {
...
+}
+
+func forEach64Values[K IntKey, V any](pairs []pair[K, V], f func(v V) bool) {
+	for _, p := range pairs {
+		if p.K != K(0) && !f(p.V) {
+			return
+		}
+	}
+}
+
```
Comment (kamstrup) [inline map64.go]: Since these 2 functions just have a single call site I would prefer if we just fold them into Keys() and Values()

**Review by kamstrup — COMMENTED** [review]
Hey thanks for your contribution! Just a minor nitpick

File: map64.go (hunk 1 above)
Comment (errx) [reply map64.go]: Thanks for the suggestion, done!

**Review by kamstrup — APPROVED** [review]
Great!

**Comment by kamstrup** [comment]
Just released 0.5.0 with this new API :+1:

**Pull request merged**